indico/modules/events/abstracts/forms.py

```diff
@@ -121,6 +121,10 @@ class AbstractSubmissionSettingsForm(IndicoForm):
                                   description=_("Allow the selection of the abstract speakers"))
     speakers_required = BooleanField(_('Require a speaker'), [HiddenUnless('allow_speakers')], widget=SwitchWidget(),
                                      description=_("Make the selection of at least one author as speaker mandatory"))
+    allow_editing = IndicoEnumSelectField(_('Allow editing'), enum=AllowEditingType, sorted=True)
+    only_speaker_submitter = BooleanField(_('Submission rights'), widget=SwitchWidget(),
```

Comment:
The naming of the option seems a bit vague, maybe? It would be nice to see how this renders, but from what I see it may be perceived as confusing. What about using an enum like in the field just before?